build.xml

```diff
@@ -365,7 +365,7 @@
           <dependency groupId="org.xerial.snappy" artifactId="snappy-java" version="1.1.10.1"/>
           <dependency groupId="net.jpountz.lz4" artifactId="lz4" version="1.3.0"/>
           <dependency groupId="com.ning" artifactId="compress-lzf" version="0.8.4"/>
-          <dependency groupId="com.google.guava" artifactId="guava" version="16.0"/>
+          <dependency groupId="com.google.guava" artifactId="guava" version="31.1-jre"/>
```

Comment:
Can we use 32.1.2-jre (latest)? Idea is that we’ll resolve up to that version from all our libraries, so it’s best to verify compatibility premerge here

Reply:
sls-cassandra is currently resolving to guava 31.1-jre. But I'm ok bumping this to latest